### ISSUE TITLE
Lists (bulleted and numbered) aren't visible when font-size is huge

### ISSUE THREAD

**Issue by Rob1kelly15** [microsoft/roosterjs#2572]
**Describe the bug**
Lists (bulleted and numbered) aren't visible when font-size is huge
**To Reproduce**
1. Make font size 48 or above. 
2. toggle bullet or numbered list.  

**Expected behavior**
you should be able to see the bullet, and it should be properly indented.  
UL should have a default padding-inline-start that equals the font-size

**Screenshots**
![image](https://github.com/microsoft/roosterjs/assets/25012942/24fee561-f2cc-486d-ab91-c63f6307b9b8)

**Device Information**
 browser - demo site

**Comment by JiuqingSong** [comment]
I think we need to increase the indent when font is large enough

**Comment by Rob1kelly15** [comment]
making it proportional to the font-size or even equal the font size would have it consistently indented for all font sizes. Every time the fontsize goes up, the indent shrinks, i would hate for it to be flush on font-size 48, but indented on 52 or something

**Comment by jvillalobos** [comment]
Yes, I think that, ideally, the left edge of the bullet marker should always be in the same horizontal position in the document. If the font gets very large, things should be pushed to the right, not the left.
